Doc/library/http.server.rst

```diff
@@ -41,6 +41,9 @@ handler.  Code to create and run the server looks like this::
    :attr:`server_port`. The server is accessible by the handler, typically
    through the handler's :attr:`server` instance variable.
 
+   .. versionchanged:: next
+      Added support for HTTP Range header.
```

Comment:
Do you think we could add some RFC section so that readers could understand the format as well?